scripts.d/s-salome-tar

```diff
@@ -29,6 +29,7 @@ function s-salome-tar-env()
 
     set-var SALOME_BIN_APPLICATION $PREFIX
     set-var SALOME_BIN_MODULES     $PREFIX/$BINDIR
+    set-var ROOT_SALOME_INSTALL    \$SALOME_ROOT_DIR/../..
```

Comment:
Why \$ here ? I would have thought that a simple "$SALOME_ROOT_DIR/../.." would work, no?

And if ok, then you can resolve the final directory directly using realpath:

set-var ROOT_SALOME_INSTALL    $(realpath $SALOME_ROOT_DIR/../..)

Reply:
I see, SALOME_ROOT_DIR is defined en env_lauch.sh and salome main driver (from binary archive). And this is needed for OpenTURNS tests to point to salome installation which is defined later relative to the salome driver. That being said I think a better solution is to generate in extra.env.d the file openturns.py:

```
# openturns
import os.path
def init(context, root_dir):
   context.setVariable(r"ROOT_SALOME_INSTALL", root_dir)
```

Can you test this?

Reply:
You can look at what is done in `s-salome-ng` in `wrapup` hook.